test/C/src/ActionDelay.lf

```diff
@@ -1,5 +1,7 @@
 // Test logical action with delay.
-target C
+target C {
+  platform: Zephyr
```

Comment:
Why is this change included in this PR? This comment also applies to the other changes to the C tests. It looks like the changes sneaked in with the "[Have TracePluginProperty simple inherit from StringProperty](https://github.com/lf-lang/lingua-franca/pull/2275/commits/aaf29395e5b4409b5a0892209fa0deb4a77c698d)" commit.

Reply:
Yes, thank you! This is a mistake